tests/imfile-wildcards-dirs-multi4.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under GPLv3
+export IMFILEINPUTFILES="1"
+export IMFILEINPUTFILESSTEPS="5"
+export IMFILEINPUTFILESALL=$(($IMFILEINPUTFILES * $IMFILEINPUTFILESSTEPS))
+echo [imfile-wildcards-multi4.sh]
+. $srcdir/diag.sh check-inotify-only
+. $srcdir/diag.sh init
+# generate input files first. Note that rsyslog processes it as
+# soon as it start up (so the file should exist at that point).
+
+# Start rsyslog now before adding more files
+. $srcdir/diag.sh startup imfile-wildcards-dirs-multi4.conf
+# sleep a little to give rsyslog a chance to begin processing
+sleep 1
+
+for i in `seq 1 $IMFILEINPUTFILES`;
+do
+	echo "Make rsyslog.input.dir$i"
+	mkdir rsyslog.input.dir$i
+	./msleep 100
```

Comment:
this needs to go away -- it will make the test not match reality

Reply:
can be deleted, also test not active yet